docs.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "strong-xstats",
```

Comment:
do we have a naming convention for express middleware? should we? maybe `express-agent-stats` or `strong-express-stats`? Even though this can be used without agent, I don't think its terrible to put agent in the name. It is the main intended use, anybody wanting to emit records to themself could do it in a few lines without this module.